account-kit/smart-contracts/src/ma-v2/abis/maV2.ts

```diff
@@ -0,0 +1,1241 @@
+export const maV2Abi = [
```

Comment:
Nit: Prefer to name this file after the contract it is from. If it is from `SemiModularAccountBytecode`, let's perhaps call it that? It seems we aren't consistent with naming between what we do for light account and msca though. Leaning towards sticking with the msca convention.